modules/generator-primer-module/app/templates/README.md

```diff
@@ -0,0 +1,63 @@
+# Primer CSS / <%= title %>
+
+[![npm version](http://img.shields.io/npm/v/<%= module %>.svg)](https://www.npmjs.org/package/<%= module %>)
+[![Build Status](https://travis-ci.org/primer/primer-css.svg?branch=master)](https://travis-ci.org/primer/primer-css)
+
+> <%= description %>
+
+This repository is a module of the full [primer-css][primer-css] repository.
+
+## Install
+
+This repository is distributed with [npm]. After [installing npm][install-npm], you can install `<%= module %>` with this command.
+
+```
+$ npm install --save <%= module %>
+```
+
+## Usage
+
+<% if (type === "css") { %>
+The source files included are written in [Sass][sass] (SCSS) You can simply point your sass `include-path` at your `node_modules` directory and import it like this.
+
+```scss
+@import "<%= module %>/index.scss";
+```
+
+You can also import specific portions of the module by importing those partials from the `/lib/` folder. _Make sure you import any requirements along with the modules._
+<% } else { %>
+<%= usage %>
+<% } %>
+
+## Build
+
+<% if (type === "css") { %>
+For a compiled **css** version of this module, a npm script is included that will output a css version to `build/build.css` The built css file is also included in the npm package.
+
+```
+$ npm run build
+```
+<% } else { %>
+<%= build %>
+<% } %>
+
+## Documentation
+
+<!-- %docs
+title: <%= title %>
+status: <%= status || "Experimental" %>
```

Comment:
`Experimental` status is probably right for new modules that we want to publish and have docs for in the style guide while we're testing them. But I think we should discuss this a bit further and figure out the relationship between status, versioning, alpha or beta releases (if we decide to do them), and if/when to include in the style guide.